mne_bids/mne_bids.py

```diff
@@ -359,7 +380,7 @@ def _sidecar_json(raw, task, manufacturer, fname, kind,
     n_miscchan = len([ch for ch in raw.info['chs']
                      if ch['kind'] == FIFF.FIFFV_MISC_CH])
     n_stimchan = len([ch for ch in raw.info['chs']
-                     if ch['kind'] == FIFF.FIFFV_STIM_CH])
+                     if ch['kind'] == FIFF.FIFFV_STIM_CH]) - num_ignored
```

Comment:
see two comments above on lines 363 and 41

Reply:
please say `n_ignored` not `num_ignored`.